src/composables/transactionTokens.ts

```diff
@@ -66,12 +66,18 @@ export function useTransactionTokens({
 
     const symbol = isAllowance ? AE_SYMBOL : getTxAssetSymbol(transaction);
     const token = protocolTokens[transaction.tx.contractId];
+    const isAe = isAllowance || (symbol === AE_SYMBOL && !isTransactionAex9(transaction));
```

Comment:
I strongly suggest to avoid any checks for specific protocol. Maybe the adapter could have a method `isTransactionCoin` and the logic for protocol specific check could be moved there?

Reply:
`isAe` seems to be heavily used. Even if the check is moved to the adapter we still need `isAe` in order to add it to the resolved token

Reply:
I created a task for refactoring this #2722